lib/main.dart

```diff
@@ -1,119 +1,98 @@
 import 'package:flutter/material.dart';
+
+import 'package:coffeecard/base/style/colors.dart';
 import 'package:coffeecard/base/style/text_styles.dart';
 
+import 'package:coffeecard/widgets/pages/tickets_page.dart';
+import 'package:coffeecard/widgets/pages/receipts_page.dart';
+import 'package:coffeecard/widgets/pages/stats_page.dart';
+import 'package:coffeecard/widgets/pages/settings_page.dart';
+
 void main() {
   runApp(MyApp());
 }
 
 class MyApp extends StatelessWidget {
-  // This widget is the root of your application.
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
-      title: 'Flutter Demo',
+      title: 'Café Analog',
```

Comment:
We're moving over to call us `Cafe Analog` without the `é`

Reply:
This should also be moved to the StringConstants as suggested in the comment for `lib/widgets/pages/tickets_page.dart`